Assets/Scripts/LazMovementController.cs

```diff
@@ -0,0 +1,113 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.InputSystem;
+
+public class LazMovementController : MonoBehaviour
+{
+    private event Action OnLazoActivate;
```

Comment:
We shouldn't be activating Lazo here. 

This class should only be for moving

Reply:
Updated